src/Native/Silk.NET.WebGPU.Native.WGPU/build/net461/Silk.NET.WebGPU.Native.WGPU.props

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_Ultz_NativePackages_WGPU_IsWindows Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">true</_Ultz_NativePackages_WGPU_IsWindows>
```

Comment:
Please don't use the Ultz name in new code.

Should be `_Silk_NET_WebGPU_Native_WGPU_` instead